.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,6 +7,8 @@ about: Create a report to help us improve
 **Describe the bug**
 A clear and concise description of what the bug is.
 
+For UI issues please also add a screenshot that shows the issue.
```

Comment:
arent the UI issues going to be open into kiali/kiali-ui?

Reply:
kiali-ui issues are disabled, all issues are going here AFAIK

Reply:
Yes, only here. One place to remember is hard enough :-)
Also users that just get Kiali via docker container and want to report a bug will see this one.